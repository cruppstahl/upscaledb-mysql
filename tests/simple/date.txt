DROP TABLE IF EXISTS test;
CREATE TABLE test (d YEAR) ENGINE=upscaledb;
INSERT INTO test VALUES ('2010');
SELECT * FROM test;
INSERT INTO test VALUES ('2011');
SELECT * FROM test;
INSERT INTO test VALUES ('2012');
SELECT * FROM test;
