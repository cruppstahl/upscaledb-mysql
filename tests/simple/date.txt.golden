--------------
DROP TABLE IF EXISTS test
--------------

--------------
CREATE TABLE test (d YEAR) ENGINE=upscaledb
--------------

--------------
INSERT INTO test VALUES ('2010')
--------------

--------------
SELECT * FROM test
--------------

d
2010
--------------
INSERT INTO test VALUES ('2011')
--------------

--------------
SELECT * FROM test
--------------

d
2010
2011
--------------
INSERT INTO test VALUES ('2012')
--------------

--------------
SELECT * FROM test
--------------

d
2010
2011
2012
